src/app-layout/index.tsx

```diff
@@ -173,60 +146,29 @@ const OldAppLayout = React.forwardRef(
       { componentName: 'AppLayout', controlledProp: 'toolsOpen', changeHandler: 'onToolsChange' }
     );
 
-    const [activeDrawerId, setActiveDrawerId] = useControllable(
-      drawers?.activeDrawerId,
-      drawers?.onChange,
-      isMobile ? false : tools ? defaults.toolsOpen : '',
-      {
-        componentName: 'AppLayout',
-        controlledProp: 'activeDrawerId',
-        changeHandler: 'onChange',
-      }
-    );
-
-    const { iconName, getLabels } = togglesConfig.tools;
-    const { mainLabel, closeLabel, openLabel } = getLabels(ariaLabels);
-
-    const toolsItem = {
-      id: 'tools',
-      content: tools,
-      resizable: false,
-      ariaLabels: {
-        triggerButton: openLabel,
-        closeButton: closeLabel,
-        content: mainLabel,
-      },
-      trigger: {
-        iconName: iconName,
-      },
-    };
-
-    const getAllDrawerItems = () => {
-      if (!hasDrawers) {
-        return;
-      }
-      return tools ? [toolsItem, ...drawers.items] : drawers.items;
-    };
-
-    const selectedDrawer =
-      tools && toolsOpen
-        ? toolsItem
-        : hasDrawers
-        ? getAllDrawerItems()?.filter((drawerItem: DrawerItem) => drawerItem.id === activeDrawerId)[0]
-        : undefined;
+    const {
+      drawers,
+      activeDrawer,
+      activeDrawerSize,
+      activeDrawerId,
+      onActiveDrawerChange,
+      onActiveDrawerResize,
+      ...drawersProps
```

Comment:
same as the other comment for the `...drawersProps` vs just using the `ariaLabel` directly

Reply:
Same answer: https://github.com/cloudscape-design/components/pull/1462#discussion_r1301621092